src/x509/asn1/tag.ts

```diff
@@ -51,7 +51,7 @@ export class ASN1Tag {
 
   public isContextSpecific(num?: number): boolean {
     const res = this.class === TAG_CLASS.CONTEXT_SPECIFIC;
-    return num ? res && this.number === num : res;
+    return num !== undefined ? res && this.number === num : res;
```

Comment:
Bug fix in `ASN1Tag.isContextSpecific`. Wasn't distinguishing between a missing `num` param and a falsy value like `0x00`